### PR TITLE
Replace level list with table in readme.md

### PR DESCRIPTION
**Pull request by pedrottimark**
First of all, thank you for this excellent package <3

To use 256 background colors when available, I tripped over 0-based versus 1-based `level`

Although `readme.md` has 0-based numbered list, here is picture of the **list** on `npm` page:

chalk-level-list

Here is picture of the proposed **table** in markdown preview of code editor:

chalk-level-table

Updated picture according to **review** comments:

chalk-level-table-2

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -132,12 +132,12 @@ If you need to change this in a reusable module, create a new instance:
...
 
-Levels are as follows:
-
-0. All colors disabled
-1. Basic color support (16 colors)
-2. 256 color support
-3. Truecolor support (16 million colors)
+| level | description |
+| ---: | :--- |
+| `0` | All colors disabled |
```
Comment (kevva) [inline readme.md]: Could you center the numbers in the level column?

File: readme.md [hunk 2]
```diff
@@ -132,12 +132,12 @@ If you need to change this in a reusable module, create a new instance:
 const ctx = new chalk.Instance({level: 0});
 ```
 
-Levels are as follows:
-
-0. All colors disabled
-1. Basic color support (16 colors)
-2. 256 color support
-3. Truecolor support (16 million colors)
+| level | description |
```
Comment (kevva) [inline readme.md]: Use capitalization. `Level` and `Description`.

File: readme.md [hunk 3]
```diff
@@ -132,12 +132,12 @@ If you need to change this in a reusable module, create a new instance:
 const ctx = new chalk.Instance({level: 0});
 ```
 
-Levels are as follows:
```
Comment (kevva) [inline readme.md]: Why did you remove this?

Comment (pedrottimark) [reply readme.md]: It seemed like the the heading row of the table communicates the same information.

I am happy to put it back if you prefer.

Comment (kevva) [reply readme.md]: Indeed, I guess they do. I don't have a strong opinion about it so let's remove it for now unless somebody else wants to keep it.

**Comment by Qix-** [comment]
Good catch, thank you :)
